data/whispering void/elenchus.txt

```diff
@@ -99,34 +99,42 @@ fleet "Devil-Run Warships"
 	names "pirate"
 	cargo 0
 	personality
-		heroic plunders harvests
-	variant 3
-		"Leviathan"
-	variant 3
-		"Leviathan (Laser)"
-	variant 3
-		"Leviathan (Heavy)"
-	variant 6
-		"Bactrian"
-		"Dagger" 3
-	variant 2
-		"Vanguard (Particle)"
-	variant 1
-		"Vanguard (Missile)"
-	variant 1
-		"Vanguard"
+		heroic plunders harvests disables
```

Comment:
In practice, the weapons these ships use aren't actually going to disable without killing anyway.
Also, going near them to save them is more likely to get them blown up than staying away.

Personally, I would rather the mission just fail if people get scattered and beaten up, so I can know and try it again efficiently, rather than beating around the bush for 20 minutes trying to rescue a doomed ally.
```suggestion
		heroic plunders harvests
```

Reply:
The ally would just not be killed as quickly (hopefully) giving more time for the player to tribute the world
But sure its your battle x)
Nobody is going to bother with healing the allies, are they?

Reply:
I saw npcs being disabled and not killed multiple times btw, I think they wouldnt have been with disable.

Reply:
When discussing in discord Grey said they should even be vindictive so:
```suggestion
		heroic plunders harvests vindictive
```
I was mostly thinking from a gameplay perspective